UndertaleModTool/Repackers/CopySpriteBgFontInternal.csx

```diff
@@ -413,7 +413,7 @@ List<string> GetSplitStringsList(string assetType)
     List<string> splitStringsList = new List<string>();
     string InputtedText = "";
     InputtedText = SimpleTextInput("Menu", "Enter the name(s) of the " + assetType + "(s)", InputtedText, true);
-    string[] IndividualLineArray = InputtedText.Split(new[] { "\n" }, StringSplitOptions.RemoveEmptyEntries);
+    string[] IndividualLineArray = InputtedText.Split(new[] { '\n', '\v' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
This case should get handled in SimpleTextInput rather than in the script itself.